.github/workflows/zizmor.yml

```
@@ -0,0 +1,27 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["**"]
```

```suggestion
  push:
    branches: ["main"]
    paths:
      - '.github/**'
  pull_request:
    paths:
      - '.github/**'
```
If we want to limit which PRs this runs on, we can check the files being changed.

We'd also want to scope the `zizmor-action` with `inputs: .github/` to make sure it's clear that's how it all wires up. By default zizmor looks at all the files.